core/src/main/java/com/linecorp/armeria/client/circuitbreaker/FailFastException.java

```diff
@@ -27,7 +31,11 @@
 
     private final CircuitBreaker circuitBreaker;
 
-    FailFastException(CircuitBreaker circuitBreaker) {
+    /**
+     * Construct an {@link FailFastException} that a request has been failed by {@code circuitBreaker}.
+     */
+    public FailFastException(@Nonnull CircuitBreaker circuitBreaker) {
```

Comment:
Armeria does not use `@Nonnull` since everything is non-null except `@Nullable`. You can check the detail definition in their `package-info.java`.
https://github.com/line/armeria/blob/master/CONTRIBUTING.md#use-nullable

Reply:
Ooops!

Thanks for pointing!